pkg/graph/schema.resolvers_test.go

```diff
@@ -155,3 +156,8 @@ func TestGraphQLTestSuite(t *testing.T) {
 
 	suite.Run(t, storeTestSuite)
 }
+
+func date(year, month, day int) *time.Time {
```

Comment:
Non-blocking nit: this func seems to be used exclusively in the test file, so I would encourage moving it there if it's not used in "production code".

Reply:
I was thinking of it as a utility that would probably get used in other test files, but I'll move it since we aren't doing that yet.